src/main/java/org/takes/rs/RsVelocity.java

```diff
@@ -131,7 +132,7 @@ public InputStream body() throws IOException {
      * @throws IOException If fails
      */
     private static InputStream render(final InputStream page,
-        final Map<String, Object> params) throws IOException {
+        final Map<CharSequence, Object> params) throws IOException {
```

Comment:
@YamStranger I am asking because you did so in `RsWithHeaders` constructor. I didn't ask you to change it yet but was looking for an explanation.